java/piranha/src/main/java/com/uber/piranha/XPFlagCleaner.java

```diff
@@ -408,9 +425,11 @@ private boolean isSymbolAndMatchesFlagName(Symbol argSym) {
    * @return True if matches. Otherwise false
    */
   private boolean isVarSymbolAndMatchesFlagName(Symbol argSym) {
-    return argSym instanceof Symbol.VarSymbol
-        && ((Symbol.VarSymbol) argSym).getConstantValue() != null
-        && ((Symbol.VarSymbol) argSym).getConstantValue().equals(xpFlagName);
+    if (argSym instanceof Symbol.VarSymbol) {
+      Object constantValue = ((Symbol.VarSymbol) argSym).getConstantValue();
+      return constantValue != null && constantValue.equals(xpFlagName);
+    }
+    return false;
```

Comment:
An example of a method that must be changed to basically equivalent code for NullAway's understanding benefit :)